formats/mainstream_browse_page/publisher/details.json

```diff
@@ -3,6 +3,9 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
+    "child_ordering": {
+      "type": "string"
```

Comment:
This schema forms the contract for what data is passed between the apps, and therefore it's important that this is as specific as needed.  Any validation in the apps is secondary.